packages/nc-gui/components/nc/SelectOption.vue

```diff
@@ -0,0 +1,23 @@
+<template>
```

Comment:
where do we use this component?

Reply:
**Removed in below commit**

[fix : removed SelectOption component](https://github.com/nocodb/nocodb/pull/7181/commits/0ef580d48e9c98ce3c4cc2e3b4cdfd37f9637860)
[0ef580d](https://github.com/nocodb/nocodb/pull/7181/commits/0ef580d48e9c98ce3c4cc2e3b4cdfd37f9637860)